TACC/tup-ui
# refactor: use core-styles v2.15, not form.cms.css

## Overview

Provide `s-form` from https://github.com/TACC/Core-Styles.

## Related

- replace `s-form` in #270 
- similar to https://github.com/TACC/Core-CMS/pull/678

## Changes

## Testing

1. `npm install` to get https://github.com/TACC/Core-Styles/releases/tag/v2.15.0.
2. Remove `.s-form` (see origin in #270).
3. Verify all MFA forms styles are uncahnged.

## UI

Pay attention to form fields and labels, not section layout and image size.

| | Before | After |
| - | - | - |
| S<br />M<br />S | ![sms - before](https://user-images.githubusercontent.com/62723358/255733304-b9797a1a-b0df-4625-a01e-f287766b4373.png) | ![sms - after](https://github.com/TACC/tup-ui/assets/62723358/6e7c7d6b-705d-471d-b469-5c6d34dfc42c)
| Q<br />R | ![qr - before](https://user-images.githubusercontent.com/62723358/255732061-9177478e-c51d-4d6c-89ce-47616ad8d2c3.png) | ![qr - after](https://github.com/TACC/tup-ui/assets/62723358/7ee89150-9fa5-45c1-9936-5b4349cee0c5)
